compiler/test/formatter_outputs/constraints.gr

```diff
@@ -0,0 +1,8 @@
+import List from "list"
+let test = test => {
+  // Comments
+  List.forEach(i => {
+    // Comment In ForEach
+    print(i)
```

Comment:
So this is just testing they **don't** move? I ask because the input and output look exactly the same to me.

Reply:
That's correct, they were moving before.